web/src/hash_util.ts

```diff
@@ -7,6 +7,7 @@ import * as people from "./people.ts";
 import * as settings_data from "./settings_data.ts";
 import type {NarrowTerm} from "./state_data.ts";
 import * as stream_data from "./stream_data.ts";
+import {get_latest_message_id_in_topic} from "./stream_topic_history.ts";
```

Comment:
We always import functions via `import *`, so that it's easier to read the code. (types and classes use a different pattern)

Reply:
Changed.